clang/test/Driver/ptrauth.c

```diff
@@ -0,0 +1,32 @@
+// Check that we can manually enable specific ptrauth features.
+
+// RUN: %clang -target aarch64 -c %s -### 2>&1 | FileCheck %s --check-prefix NONE
```

Comment:
`--target=` for new tests

Reply:
Fixed, thanks, see f80f4bd89a316611a507e79c029e64065681caf6